shared/devices/index.js

```diff
@@ -35,7 +35,8 @@ class Devices extends Component {
   render () {
     return (
       <Render
-        devices={this.props.devices}
+        devices={this.props.devices && this.props.devices.filter(dev => !dev.revokedAt)}
+        revokedDevices={this.props.devices && this.props.devices.filter(dev => dev.revokedAt)}
         waitingForServer={this.props.waitingForServer}
```

Comment:
The efficiency-purist in me would like a:
```js
{devices, revokedDevices} = this.props.devices.splitOn(dev => dev.revokedAt)
```
func that makes both arrays at once.  :)  Ah!  Found one:

https://lodash.com/docs#partition

Shall I use it?